.github/workflows/docker-build.yaml

```
@@ -0,0 +1,14 @@
+name: Docker Image CI
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
```

Not sure if you don't have to add the branch name?
```suggestion
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

---

'cause I would expect it to run on this PR with what you currently have :sweat_smile: